content/doc/book/managing/spawning-processes.adoc

```diff
@@ -53,7 +53,7 @@ daemonize -E BUILD_ID=dontKillMe /path/to/your/command
 In a Jenkins Pipeline, use `+JENKINS_NODE_COOKIE+` instead of `+BUILD_ID+`.
 
 Note that this will set the BUILD_ID environment variable for the process being spawned to something other than the current BUILD_ID.
-Or you can start jenkins with `-Dhudson.util.ProcessTree.disable=true` - see link:/doc/book/using/long-running-agent-process[long running agent process] for details.
+Or you can start jenkins with `-Dhudson.util.ProcessTree.disable=true` - see link:/doc/book/using/using-agents/[long running agent process] for details.
```

Comment:
```suggestion
Or you can start jenkins with `-Dhudson.util.ProcessTree.disable=true` - see link:/doc/book/using/using-agents/[long running agent process] for details.
```
I don't believe that this link would be the same content as what was originally here. If there is a specific section on that page that would be more relevant, it would be best to use that direct link. If there is not relevant content on that page, consider removing the link in this case.

Reply:
There is no relevant content on that page, I will go ahead and remove the link in this case then :)